### PR TITLE
Upgrade PublishPipelineMetadataV0 to NodeJS16

### PR DESCRIPTION
**Pull request by sergey-koryshev**
**Task name**: PublishPipelineMetadataV0

**Description**: This PR upgrades PublishPipelineMetadataV0 to NodeJS 16

**Documentation changes required:** (Y/N) N

**Added unit tests:** (Y/N) N

**Attached related issue:** (Y/N) N

**Checklist**:
- [x] Task version was bumped - please check [instruction](https://github.com/microsoft/azure-pipelines-tasks/tree/master/docs/taskversionbumping.md) how to do it
- [x] Checked that applied changes work as expected

### CODE REVIEW

**Review by Roman-Shchukin — APPROVED** [review]

**Review by ismayilov-ismayil — APPROVED** [review]

File: Tasks/PublishPipelineMetadataV0/package-lock.json [hunk 1]
```diff
@@ -1,8 +1,721 @@
 {
   "name": "publishpipelinemetadatav0",
   "version": "1.0.0",
-  "lockfileVersion": 1,
+  "lockfileVersion": 2,
```
Comment (vmapetr) [inline Tasks/PublishPipelineMetadataV0/package-lock.json:4]: https://github.com/microsoft/azure-pipelines-tasks/pull/17277#discussion_r1032273934
I guess the same here.

Comment (sergey-koryshev) [reply Tasks/PublishPipelineMetadataV0/package-lock.json]: Fixed

**Review by v-mohithgc — APPROVED** [review]
Tested, please change the task minor version to 216. Thanks

**Comment by sergey-koryshev** [comment]
@v-mohithgc, thank you for reviewing, I will update version number to actual one before PR is merged.

**Review by rvairavelu — APPROVED** [review]
approved

**Comment by v-mohithgc** [comment]
@sergey-koryshev can you please change task minor version to 216?

**Review by kirill-ivlev — APPROVED** [review]

**Pull request merged**